package-lock.json

```diff
@@ -14311,8 +14311,8 @@
       "integrity": "sha512-6qE4B9deFBIa9YSpOc9O0Sgc43zTeVYbgDT5veRKSlB2+ZuHNoVVxA1L/ckMUayV9Ay9y7Z/SZCLcGteW9i7bg=="
     },
     "mattermost-redux": {
-      "version": "github:mattermost/mattermost-redux#6704b3516be0d0fa1f00f1d472719f3f236e9fb9",
-      "from": "github:mattermost/mattermost-redux#6704b3516be0d0fa1f00f1d472719f3f236e9fb9",
+      "version": "github:mattermost/mattermost-redux#0935ea32bf849339ecabab2e039d160e4ddacbae",
```

Comment:
This is the commit from the release-5.24 branch of redux

https://github.com/mattermost/mattermost-redux/commit/0935ea32bf849339ecabab2e039d160e4ddacbae